src/atomate2/qchem/jobs/base.py

```diff
@@ -0,0 +1,158 @@
+"""Definition of a base QChem Maker"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Callable
+
+from jobflow import Maker, Response, job
+from monty.serialization import dumpfn
+from monty.shutil import gzip_dir
+
+from pymatgen.core.structure import Molecule
+from pymatgen.io.qchem.inputs import QCInput
+
+from atomate2.qchem.files import copy_qchem_outputs, write_qchem_input_set
+from atomate2.qchem.run import run_qchem, should_stop_children
+from atomate2.qchem.schemas.task import TaskDocument
+from atomate2.qchem.sets.base import QChemInputGenerator
+
+__all__ = ["BaseQChemMaker", "qchem_job"]
+
+# _DATA_OBJECTS = [
+#     BandStructure,
+#     BandStructureSymmLine,
+#     DOS,
+#     Dos,
+#     CompleteDos,
+#     Locpot,
+#     Chgcar,
+#     Wavecar,
+#     Trajectory,
+#     "force_constants",
+#     "normalmode_eigenvecs",
+# ]
+
+_DATA_OBJECTS = [QCInput]
```

Comment:
DATA_OBJECTS controls what objects will get put into the jobflow "data" store which is used for large objects. I don't think QCInput should be in here as:
1. QCInputs won't be getting stored after a calculations.
2. They aren't large!

Reply:
Agreed. This field would be for large data files, for example if we write out a CUBE file of the electron density or something like that.